integration/generic-tests/universalpayload_amd64_test.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2025 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build amd64 && !race
+
+package integration
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/Netflix/go-expect"
+	"github.com/hugelgupf/vmtest/qemu"
+	"github.com/hugelgupf/vmtest/scriptvm"
+	"github.com/u-root/mkuimage/uimage"
+)
+
+// TestUPLBootAmd64 tests '/bbin/kexec UplFitX64.fit' to boot to UEFI Shell.
+func TestUPLBootAmd64(t *testing.T) {
+	//os.Setenv("VMTEST_ARCH", "amd64")
```

Comment:
Can we omit this comment or did you place it here on purpose?

Reply:
Sorry, forget to remove it. Let me remove it.